sources/Vulkan/Vulkan/Handles/BufferTHandle.gen.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// Ported from the Vulkan headers and corresponding dependencies.
+// Original source is Copyright 2015-2023 The Khronos Group Inc. Licensed under the MIT license.
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Silk.NET.Vulkan;
+
+public readonly unsafe partial struct BufferTHandle
```

Comment:
> Because this has been reverted, the current output is BufferTHandle, but that will be fixed with the new renamer approach (save native name early on and replace native name at end

This is in the "done" section, but the committed bindings still have the T suffix. Is this going to be fixed in a future PR?